src/msal-client-credentials-flow/README.md

```diff
@@ -0,0 +1,170 @@
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Configure the web API
+
+1. Open the _~/msal-client-credentials-flow/appsettings.json_ file in your code editor and modify the following values values with those from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis):
+
+   ```json
+   "ClientId": "Enter_the_Application_Id_here",
+   "TenantId": "Enter_the_Tenant_Info_Here",
+   "ClientSecret": "Enter_the_Application_CLient_Secret_Here",
+   ...
+   "RelativePath": "Enter_the_Application_Object_Id_Here",
+   ...
+   ```
+
+## Run the application
+
+### 1. Run the web API
+
+1. Execute the following command to get the app up and running:
+
+   ```bash
+   dotnet run
+   ```
+
+### 2. Send request to the web API
+
+1. Once the app is listening, execute the following to send the a request.
+
+   ```bash
+   curl -X GET https://localhost:5001/application -ki
+   ```
+
+   :information_source: Since the request is sent without a Bearer Token, it is expected to receive an Unauthorized response `401`. The web API is now protected
+
+   If everything worked, you should receive a response from the downstream web API (Microsoft Graph, in this case) similar to this:
+
+   ```console
+   $ curl https://localhost:5001/api/application -ki
+   {
+     "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#applications/$entity",
+     "id": "003c4673-d4a0-4eb6-8979-d6269e08b87e",
+     "deletedDateTime": null,
+     "appId": "c085fc22-2484-41c0-b795-2e940f51df64",
+     "applicationTemplateId": null,
+     "disabledByMicrosoftStatus": null,
+     "createdDateTime": "2021-12-17T16:45:43Z",
+     "displayName": "active-directory-dotnet-minimal-api-aspnetcore-client-credentail-flow",
+     "description": null,
+     "groupMembershipClaims": null,
+     "identifierUris": ["api://c085fc22-2484-41c0-b795-2e940f51df64"],
+     "isDeviceOnlyAuthSupported": null,
+     "isFallbackPublicClient": true,
+     "notes": null,
+     "publisherDomain": "contoso.onmicrosoft.com",
+     "serviceManagementReference": null,
+     "signInAudience": "AzureADMyOrg",
+     "tags": [],
+     "tokenEncryptionKeyId": null,
+     "defaultRedirectUri": null,
+     "certification": null,
+     "optionalClaims": null
+     ...
+   }
+   ```
+
+## About the code
+
+This ASP.NET Core minimal web API has a single route (_/application_) that supports anonymous access.  When the anonymous route is called, the API requests its own application object from Microsoft Graph.
+
+The Microsoft Graph response data is included in its response back to the original caller.
+
+This web API uses the following:
+
+_ [Microsoft Authentication Library (MSAL)](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-overview)
```

Comment:
Did you want to link to the ASP.NET core version instead of the MSAL conceptual home?

```suggestion
_ [Microsoft Authentication Library (MSAL)](https://docs.microsoft.com/azure/active-directory/develop/msal-overview)
```

Reply:
thanks for asking @ckittel, I think I really wanted to link that specific doc. Im proposing another alternative in here in case this is preferable: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet.

partially done | addressed from [896f3](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/22/commits/896f3495aafd1e0519a58cad4a31174ea82b1223)

Reply:
the latter might be good, since that's literally the item you're using here.  We typically don't really list stuff in this section like this.  So all kinda new to me to see a list like this with one item.  Can you just bake these links in as inline links in some text around it to that you don't need to dedicate vertical space for it?

Reply:
using latter as that is the preference + inlined it @ckittel 

done | addressed from [1439c](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/22/commits/1439c86393d2934a295fc01e7a76f384e2f9f6cf) and [1eef38](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/22/commits/1eef389e5d821487490e725eb320206885a04a7b)

Reply:
Simple, good choice.